src/frontend/apps/standalone_site/src/features/Profile/features/AccountSettings/components/PrivateTextInputField.tsx

```diff
@@ -0,0 +1,38 @@
+import { Button, FormField, TextInput } from 'grommet';
+import { useState } from 'react';
+
+import { ReactComponent as HideContent } from 'assets/svg/hide-content.svg';
+import { ReactComponent as ShowContent } from 'assets/svg/show-content.svg';
+
+interface PrivateTextInputFieldProps {
+  id: string;
+  name: string;
+  label: string;
+}
+
+export const PrivateTextInputField = ({
```

Comment:
There is an issue with this component behavior. When I have the focus in the input and I tabulate I expect to jump to the next input but it takes the focus on the `show`/`hide` logo

Reply:
But how do you expect to be able to focus the show/hide button if it is not after tabulate when the related input is focused ?

Reply:
Good point.